pkg/cmd/cluster/cluster.go

```diff
@@ -13,6 +13,7 @@ func NewClusterCommand(f *factory.Factory) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     f.Localizer.MustLocalize("cluster.cmd.use"),
 		Short:   f.Localizer.MustLocalize("cluster.cmd.shortDescription"),
+		Long:    f.Localizer.MustLocalize("cluster.cmd.longDescription"),
```

Comment:
I'm not sure if a long description is always necessary, if the short description is already clear enough. Sometimes there is not much extra to say - especially common with a parent command like `cluster`, `kafka`, `service-registry` etc.

Reply:
Short is used in the list of commands.
Long is used for specific command. When long is not present short is used but.. They should be different. 
Long one should present command memonic including positional:

```
rhoas whatever [arguments] <name> 
```
 
>  cluster, kafka, service-registry etc.

We need to add some explanation of the commands as part of those. 

For example Kafka:

## Commands for managing Kafkas

...

## Commands to interact with your instance

...